### ISSUE TITLE
Adding dummy data fails with "no such table accounts_user"

### ISSUE THREAD

**Issue by SwitchedToGitlab** [AmirAhrari/django-react-ecommerce#18]
###TLDR: 

Getting the error `sqlite3.OperationalError: no such table: accounts_user` running `add_dummy_data.py`

###Issue:

Hello! I'm trying to use this repo for a quick and dirty Django + React tutorial of sorts, and I'm running into an issue on setup. I'm sure this is user error, but I cant quite figure out why. Followed the steps on the README.md, but when running the `add_dummy_data.py` script I'm getting the error `sqlite3.OperationalError: no such table: accounts_user`. I have run `python3 manage.py makemigrations`, which completed without error (first pasted code block).

Any assistance would be sincerely appreciated. Thanks in advance!

(Complete traceback is listed below.)

### Installation environment:
```
ec2-user@ip-172-26-5-253:~> cat /etc/os-release 
NAME="openSUSE Leap"
VERSION="15.2"
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.2"
PRETTY_NAME="openSUSE Leap 15.2"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.2"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
ec2-user@ip-172-26-5-253:~> 
```

```
ec2-user@ip-172-26-5-253:~> python3 --version
Python 3.6.12
```

```
c2-user@ip-172-26-5-253:~> sqlite3 --version
3.28.0 2019-04-16 19:49:53 884b4b7e502b4e991677b53971277adfaf0a04a284f8e483e2553d0f8315alt2
```

### MIgration Output:
```
ec2-user@ip-172-26-5-253:~/django-react-ecommerce> python3 manage.py migrate
Operations to perform:
  Apply all migrations: accounts, admin, auth, carts, contenttypes, django_rest_passwordreset, orders, products, profiles, sessions
Running migrations:
  Applying accounts.0001_initial... OK
  Applying contenttypes.0001_initial... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying admin.0003_logentry_add_action_flag_choices... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0001_initial... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying auth.0008_alter_user_username_max_length... OK
  Applying auth.0009_alter_user_last_name_max_length... OK
  Applying auth.0010_alter_group_name_max_length... OK
  Applying auth.0011_update_proxy_permissions... OK
  Applying products.0001_initial... OK
  Applying carts.0001_initial... OK
  Applying django_rest_passwordreset.0001_initial... OK
  Applying django_rest_passwordreset.0002_pk_migration... OK
  Applying django_rest_passwordreset.0003_allow_blank_and_null_fields... OK
  Applying orders.0001_initial... OK
  Applying profiles.0001_initial... OK
  Applying sessions.0001_initial... OK
```

### `add_dummy_data.py` traceback:

```
ec2-user@ip-172-26-5-253:~/django-react-ecommerce> python3 add_dummy_data.py 
  File "manage.py", line 16
    ) from exc
         ^
SyntaxError: invalid syntax
Database reseted.
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 383, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: accounts_user

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "add_dummy_data.py", line 89, in <module>
    add_users()
  File "add_dummy_data.py", line 28, in add_users
    User.objects.create_user(**user)
  File "/home/ec2-user/django-react-ecommerce/accounts/models.py", line 23, in create_user
    user.save(using=self._db)
  File "/usr/lib/python3.6/site-packages/django/contrib/auth/base_user.py", line 66, in save
    super().save(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/django/db/models/base.py", line 741, in save
    force_update=force_update, update_fields=update_fields)
  File "/usr/lib/python3.6/site-packages/django/db/models/base.py", line 779, in save_base
    force_update, using, update_fields,
  File "/usr/lib/python3.6/site-packages/django/db/models/base.py", line 870, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/usr/lib/python3.6/site-packages/django/db/models/base.py", line 908, in _do_insert
    using=using, raw=raw)
  File "/usr/lib/python3.6/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/django/db/models/query.py", line 1186, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/usr/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1368, in execute_sql
    cursor.execute(sql, params)
  File "/usr/lib/python3.6/site-packages/django/db/backends/utils.py", line 99, in execute
    return super().execute(sql, params)
  File "/usr/lib/python3.6/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/usr/lib/python3.6/site-packages/django/db/backends/utils.py", line 76, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/usr/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/lib/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 383, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: accounts_user
